cookbooks/aws-parallelcluster-config/recipes/directory_service.rb

```diff
@@ -49,6 +49,24 @@
     command "cp #{sssd_conf_path} #{shared_sssd_conf_path}"
     sensitive true
   end
+
+  if node['cluster']["directory_service"]["generate_ssh_keys_for_users"] == 'true'
+    generate_ssh_key_path = "#{node['cluster']['scripts_dir']}/generate_ssh_key.sh"
+    template generate_ssh_key_path do
+      source 'directory_service/generate_ssh_key.sh.erb'
+      owner 'root'
+      group 'root'
+      mode '0755'
+    end
+    filter_lines 'Configure PAM postlogin script to call generate SSH key script' do
```

Comment:
minor: name is wrong

Reply:
should probably be 'PAM sshd config' instead of PAM postlogin script

Reply:
Done